packages/terra-time-input/src/orion-fusion-theme/TimeInput.module.scss

```diff
@@ -39,6 +39,8 @@
     --terra-time-input-focus-ltr-box-shadow: 0 0 8px #a6d9f4;
     --terra-time-input-focus-rtl-box-shadow: 0 0 8px #a6d9f4;
     --terra-time-input-focus-color: #1c1f21;
+    --terra-time-input-outline: 2px dashed #000;
+    --terra-time-input-meridiem-button-focus-border-outline: 2px dashed #000;
```

Comment:
We have not used #000 as outline color for low-light theme I think you need compare this value with other components which has got the outline styles.